src/map.cpp

```diff
@@ -620,7 +620,8 @@ vehicle *map::move_vehicle( vehicle &veh, const tripoint &dp, const tileray &fac
             veh.handle_trap( wheel_p, w );
             if( !has_flag( "SEALED", wheel_p ) ) {
                 // TODO: Make this value depend on the wheel
-                smash_items( wheel_p, 5 );
+                //~ %1$s: vehicle name
+                smash_items( wheel_p, 5, string_format( _( "weight of %1$s" ), veh.disp_name() ) );
```

Comment:
There's nothing to format here.

```suggestion
                smash_items( wheel_p, 5, _( "weight of %1$s" ), veh.disp_name() );
```

Reply:
There's an argument (the vehicle name).

Reply:
This doesn't work. It adds a 4th parameter which `map::smash_items` doesn't know what to do with.

Reply:
Oh, my bad. Just put both 3rd and 4th arguments in one brackets, that should do it.

Reply:
That seems to cause issues as well. I wasn't sure exactly how you wanted me to group it, so I tried the two ways that were obvious to me.  

When written as `smash_items( wheel_p, 5, (_( "weight of %1$s" ), veh.disp_name()) );`, the messages start with (when using a Road roller) "The the Road Roller..."

When written as `smash_items( wheel_p, 5, _( "weight of %1$s" , veh.disp_name()) );` it throws an error when compiling: `error: no matching function for call to '_(const char [15], std::string)'`

Sorry for the trouble, but I don't have much c++ experience, nor do I understand what this change is meant to accomplish, so I'm stumbling around in the dark a bit.

Reply:
Oh, if it's causing much trouble, don't mind me. It's good and working now.